sherpa/astro/xspec/utils.py

```diff
@@ -93,4 +95,5 @@ class version_at_least(include_if):
     the xspec version detected at runtime is equal or greater than the one provided to the decorator.
     """
     def __init__(self, version_string):
-        include_if.__init__(self, _xspec.get_xsversion() >= version_string)
+        xspec_version = LooseVersion(_xspec.get_xsversion())
```

Comment:
Would it make sense to make `xspec_version` a module-level "constant", since it shouldn't be something that changes within a test run?

Reply:
Done.